Meta/Lagom/Fuzzers/AudioFuzzerCommon.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022, Luke Wilde <lukew@serenityos.org>
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+ * Copyright (c) 2021-2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Concepts.h>
+#include <AK/MemoryStream.h>
+#include <LibAudio/Loader.h>
+#include <stddef.h>
+#include <stdint.h>
+
+template<typename LoaderPluginType>
+requires(IsBaseOf<Audio::LoaderPlugin, LoaderPluginType>)
+int fuzz_audio_loader(uint8_t const* data, size_t size)
+{
+    auto const bytes = ByteBuffer::copy(data, size).release_value();
```

Comment:
I know this is how the code worked before, but could we just construct a ReadonlyBytes instead of a full ByteBuffer? That would avoid a potentially hefty copy.